src/core/Akka.Tests/Loggers/LogFilterEvaluatorSpecs.cs

```diff
@@ -0,0 +1,224 @@
+﻿// -----------------------------------------------------------------------
+//  <copyright file="LogFilterEvaluatorSpecs.cs" company="Akka.NET Project">
+//      Copyright (C) 2009-2024 Lightbend Inc. <http://www.lightbend.com>
+//      Copyright (C) 2013-2024 .NET Foundation <https://github.com/akkadotnet/akka.net>
+//  </copyright>
+// -----------------------------------------------------------------------
+
+using System.Collections.Generic;
+using System.Text.RegularExpressions;
+using FluentAssertions;
+using Akka.Actor;
+using Akka.Actor.Setup;
+using Akka.Configuration;
+using Akka.Event;
+using Akka.TestKit;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace Akka.Tests.Loggers;
+
+/// <summary>
+/// Goal of these specs are mostly to make sure our default <see cref="Regex"/>s are working as expected
+/// </summary>
+// ReSharper disable once ClassNeverInstantiated.Global
+public class LogFilterEvaluatorSpecs
+{
+    public class LogFilterSetupSpecs : AkkaSpec
```

Comment:
End to end specs - I can't easily intercept the default output from the `StandardOutWriter` so I'm swapping in my own `MinimalLogger` implementation and testing the results using that at the end. Not 100% clean, but close enough.